adapters/InMobi/InMobiAdapter/GADMediationAdapterInMobi.m

```diff
@@ -14,16 +14,27 @@
 
 #import "GADMediationAdapterInMobi.h"
 #import "GADInMobiExtras.h"
-#import "GADMAdapterInMobi.h"
 #import "GADMAdapterInMobiConstants.h"
 #import "GADMAdapterInMobiInitializer.h"
 #import "GADMAdapterInMobiRewardedAd.h"
 #import "GADMAdapterInMobiUtils.h"
 #import "GADMInMobiConsent.h"
+#import "GADMAdapterInMobiUnifiedNativeAd.h"
+#import "GADMAdapterInMobiInterstitialAd.h"
+#import "GADMAdapterInMobiBannerAd.h"
 
 @implementation GADMediationAdapterInMobi {
-  /// InMobi rewarded ad wrapper.
-  GADMAdapterInMobiRewardedAd *_rewardedAd;
+    /// InMobi rewarded ad wrapper.
+    GADMAdapterInMobiRewardedAd *_rewardedAd;
+    
+    /// InMobi banner ad object.
+    GADMAdapterInMobiBannerAd *_bannerAd;
+    
+    /// InMobi interstitial ad object.
+    GADMAdapterInMobiInterstitialAd *_interstitialAd;
+    
+    /// Google Mobile Ads unified native ad wrapper.
+    GADMAdapterInMobiUnifiedNativeAd *_nativeAd;
```

Comment:
The comments should be consistent throughout all the ad formats, we can just do:
```
/// InMobi <format> ad wrapper.
```